### DISCUSSION TITLE
ETA for v2.3.6 release?

### DISCUSSION THREAD

**Discussion by mjeveritt** [Chatterino/chatterino2#3853] [Q&A] [answered]
It's been a couple of months since the 2.3.5 release, and it looks like there are a bunch of useful commits in the master branch - is there a plan for a release date yet?

**Comment by Felanbird** [comment]
No there's no planned time for stable releases, it's whenever fourtf feels like it. If you want the new commits just use [nightly](https://github.com/Chatterino/chatterino2/releases/tag/nightly-build).

**Comment by Felanbird** [comment, marked as answer]
@mjeveritt Next week - you can get 2.4.0-beta now if you toggle on `Receive beta updates` in settings.
